subprojects/core/src/main/java/org/gradle/deployment/internal/ContinuousExecutionGate.java

```diff
@@ -14,12 +14,9 @@
  * limitations under the License.
  */
 
-package org.gradle.initialization;
-
-import javax.annotation.Nonnull;
+package org.gradle.deployment.internal;
```

Comment:
Are in `org.gradle.deployment.internal` package any _Continous build_ specific classes that would deserve it's own `org.gradle.deployment.internal.continous` package?

Reply:
The deployment API we have "naturally" supports continuous build, so there isn't necessary a separate part of it which should go in its own package. There are some parts of the API which are used by continuous build to determine whether a new build can already trigger or not, and continuous build informs the deployments whether there are changes they need to incorporate (aka redeploy). So I wouldn't split out a separate package now.